### ISSUE TITLE
Failure in the `gcov` processing `find` command, whose test is AWOL

### ISSUE THREAD

**Issue by fish2000** [codecov/codecov-python#247]
The `gcov` processing command issued by `codecov` contains malformed `find` predicates, and always fails. The failure looks like this:

![Screen Shot 2020-04-08 at 1 45 10 AM](https://user-images.githubusercontent.com/378969/78748792-dceff980-793a-11ea-84c6-c059c0c836c4.jpg)

… this is on macOS, which offers BSD `find` as the default. I verified the failure by [installing GNU `findutils` with Homebrew](https://formulae.brew.sh/formula/findutils#default) and changing a local clone of `codecov` to use that (as `/usr/local/bin/gfind`) – the error message is slightly more helpful:

```bash
`/usr/local/bin/gfind /Users/fish/Dropbox/CLU/clu -not -path './bower_components/**' -not -path './node_modules/**' -not -path './vendor/**' -type f -name *.gcno  -exec gcov -pb  {} +` exited with status 1, error: “gfind: paths must precede expression: `'”
```

… the relevant `gcov`-related test is, I think, the only test in the `codecov` suite that is [completely commented out and skipped](https://github.com/codecov/codecov-python/blob/800d1ad035e65987b094781bb328fb06e1c0e8a1/tests/test.py#L250-L259):

```python
    def test_gcov(self):
        self.skipTest("Need to fix this test...")
        # if self._env.get('TRAVIS') == 'true':
        #     self.write_c()
        #     output = self.run_cli(token='a', branch='b', commit='c')
        #     self.assertEqual(os.path.exists('hello.c.gcov'), True)
        #     report = output['reports'].split('<<<<<< network\n')[1].splitlines()
        #     self.assertIn('hello.c.gcov', report[0])
        # else:
        #     self.skipTest("Skipped, works on Travis only.")
```

… which suggests at least an acknowledgement of the problem, if not a genesis of same.

**Comment by Luttik** [comment]
I also have the gcov find issue on github actions.

**Comment by sineverba** [comment]
I can confirm this bug and with v2.0.15 it works. Related also to this issue: https://github.com/codecov/codecov-python/issues/249.

**Comment by thomasrockhu** [comment]
We recently released `2.1.3` which pulls in #217.
